susemanager-branding-oss/susemanager-branding-oss.spec

```
@@ -68,6 +70,9 @@ install -m 644 license.txt $RPM_BUILD_ROOT/%_defaultdocdir/susemanager/
 %docdir %_defaultdocdir/susemanager
 %dir %_defaultdocdir/susemanager
 %_defaultdocdir/susemanager/license.txt
+%dir %{susemanager_shared_path}
+%dir %{wwwroot}
+%dir %{wwwdocroot}
```

do we need a conditional here (I think in this way it will still fail to build on EL)?
```suggestion
%if 0%{?suse_version}
%dir %{susemanager_shared_path}
%dir %{wwwroot}
%endif
%dir %{wwwdocroot}
```